packages/augur-ui/src/modules/modal/transactions.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState, useRef } from "react";
```

Comment:
don't refactor transactions file in this PR

Reply:
Will revert this ASAP..